app/errors/routes.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python3
+from app import mongo
+from flask import Blueprint, render_template
+
+errors = Blueprint("errors", __name__)
+
+# database collection variables
+recipes_collection = mongo.db.recipes
+
+#----- Global Helper -----#
+def get_total_recipes():
+        return int(recipes_collection.count())
+@errors.context_processor
+def total_recipes():
+        return dict(total_recipes=get_total_recipes)

```

Comment:
You don't need two functions to achieve this... create a variable inside `total_recipes()`  which will take the return value of `get_total_recipe()`

Reply:
Took some time, but I finally got it sorted. FYI: this was originally a global helper variable in my single file, but with Blueprints I couldn't get it to work by itself, so I had to add it duplicated into each @app.context_process for each Blueprint file.